app/src/main/java/de/danoeh/antennapod/ui/screen/feed/SingleFeedSortDialog.java

```diff
@@ -16,11 +16,12 @@ public static SingleFeedSortDialog newInstance(Feed feed) {
         bundle.putLong(ARG_FEED_ID, feed.getId());
         bundle.putBoolean(ARG_FEED_IS_LOCAL, feed.isLocalFeed());
         if (feed.getSortOrder() == null) {
-            bundle.putString(ARG_SORT_ORDER, String.valueOf(SortOrder.DATE_NEW_OLD.code));
+            bundle.putString(ARG_SORT_ORDER, null);
```

Comment:
I'm not 100% sure about `null` vs adding a new setting `SortOrder.GLOBAL`. The new setting would make it a bit more clear for future readers of the code